tcms/testruns/tests/test_views.py

```diff
@@ -122,6 +122,30 @@ def test_show_create_new_run_page(self):
                 '<a href="%s">TC-%d: %s</a>' % (case_url, case.pk, case.summary),
                 html=True)
 
+    def test_should_be_able_to_create_new_run(self):
+        new_run_summary = 'TestRun summary'
+        post_data = {
+            'summary': new_run_summary,
+            'plan': self.plan.pk,
+            'product_id': self.plan.product_id,
+            'product': self.plan.product_id,
+            'product_version': self.plan.product_version.pk,
+            'build': self.build.pk,
```

Comment:
self.build is used only here so move the creation of this object in setup of this test class instead of in the base class. Everything else which inherits from BasePlanCase doesn't need this attribute.